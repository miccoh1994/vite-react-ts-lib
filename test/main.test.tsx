import { test, expect } from "vitest";
import renderer from 'react-test-renderer'
import {MyEpicLibraryComponent} from '@mynamespace/myepiclib';
function toJson(component: renderer.ReactTestRenderer) {
  const result = component.toJSON()
  expect(result).toBeDefined()
  expect(result).not.toBeInstanceOf(Array)
  return result as renderer.ReactTestRendererJSON
}

test('MyEpicLibraryComponent renders succesfully', () => {
  const component = renderer.create(
    <MyEpicLibraryComponent/>,
  )
  const tree = toJson(component)
  expect(tree).toMatchSnapshot()
})