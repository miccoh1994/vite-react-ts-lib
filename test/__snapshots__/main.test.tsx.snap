// Vitest Snapshot v1

exports[`MyEpicLibraryComponent renders succesfully 1`] = `
<div>
  Epic
</div>
`;

exports[`MyEpicLibraryComponent renders succesfully 2`] = `
<div>
  Epic
</div>
`;
